reconcile/prometheus_rules_tester.py

```diff
@@ -155,10 +164,23 @@ def check_valid_services(rule):
     return CommandExecutionResult(True, "")
 
 
+# We return here a CommandExecutionResult as it is what check_rule function has to
+# add to the "check_result" field.
+def check_rule_length(rule_length: int) -> CommandExecutionResult:
+    if rule_length > MAX_CONFIGMAP_SIZE:
+        return CommandExecutionResult(
+            False, f"Rule larger than {MAX_CONFIGMAP_SIZE} bytes."
```

Comment:
I think the message could be a bit confusing. I understand the problem is related to all the rules within the configmap (Alerting or recording rules) not a single rule. This could be an already existing typo as `check_rule` also seems to check the whole spec, not a single rule. 

```suggestion
            False, f"Rules spec is larger than {MAX_CONFIGMAP_SIZE} bytes."
```

Reply:
I like it. Thx!